xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h

```diff
@@ -576,6 +576,16 @@ extern "C"
    */
   bool CanSeekStream();
 
+  /*
+   * Check if the given EPG tag can be recorded.
+   * @param tag the eog tag
+   * @param the returned value, true if the tag can be recorded
+   * @return true if the tag can be recored.
+   * @remarks Optional, return PVR_ERROR_NOT_IMPLEMENTED to let kodi decide base on the current time
```

Comment:
Please remove "base on the current time", because from the api perspective of view oit does not matter what kodi makes out of this.